src/core/trtserver.h

```diff
@@ -280,6 +287,23 @@ TRTSERVER_EXPORT TRTSERVER_Error* TRTSERVER_ServerOptionsSetServerId(
 TRTSERVER_EXPORT TRTSERVER_Error* TRTSERVER_ServerOptionsSetModelRepositoryPath(
     TRTSERVER_ServerOptions* options, const char* model_repository_path);
 
+// Set the model control mode in a server options. For each mode the models
+// will be managed as the following:
+//
+//   TRTSERVER_MODEL_CONTROL_NONE: the models in model repository will be loaded
+//   on startup. The model states will not be modified after startup.
+//
+//   TRTSERVER_MODEL_CONTROL_POLL: the models in model repository will be loaded
+//   on startup. The model repository will be polled periodically and the models
```

Comment:
"The model repository can be polled periodically using TRTSERVER_ServerPollModelRepository and the server will load, unload, and updated models according to ..."